reference/test/matrix/sparsity_csr_kernels.cpp

```diff
@@ -207,6 +221,22 @@ TYPED_TEST(SparsityCsr, AppliesLinearCombinationToDenseVector)
 }
 
 
+TYPED_TEST(SparsityCsr, AppliesLinearCombinationToMixedDenseVector)
+{
+    using Vec = typename TestFixture::Vec;
+    using T = typename TestFixture::value_type;
```

Comment:
```suggestion
    using value_type = typename TestFixture::value_type;
    using T = gko::next_precision<value_type>;
    using Vec = gko::matrix::Dense<T>;
```
assuming you want `T` to be the new type.